improver/cli/vicinity.py

```diff
@@ -36,11 +41,17 @@ def process(cube: cli.inputcube, vicinity: cli.comma_separated_list = None):
             List of distances in metres used to define the vicinities within
             which to search for an occurrence. Each vicinity provided will
             lead to a different gridded field.
+        land_mask (iris.cube.Cube):
```

Comment:
You could move the description of `land_mask` up to match the order of the arguments.